src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -64,16 +57,31 @@ const Body = BaseFormWithPolling.extend(
         this.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
         break;
       case Enums.CUSTOM_URI_CHALLENGE:
-        this.title = loc('oktaVerify.button', 'login');
+        this.title = loc('customUri.title', 'login');
         this.add(View.extend({
           className: 'skinny-content',
           template: hbs`
             <p>
               {{{i18n code="customUri.required.content.p1" bundle="login"}}}
             </p>
-            <p>
-              {{{i18n code="customUri.required.content.p2" bundle="login" arguments="downloadOVLink"}}}
-            </p>
+          `,
+        }));
+        this.add(createButton({
+          className: 'ul-button button button-wide button-primary',
+          title: loc('customUri.required.content.button', 'login'),
+          click: () => {
+            this.doCustomURI();
+          }
+        }));
+        this.add(View.extend({
+          className: 'skinny-content',
+          template: hbs`
+          <p>
+            {{{i18n code="customUri.required.content.p2" bundle="login"}}}
```

Comment:
You shouldn't need to have Handlebars **not** escape this value.

```suggestion
            {{i18n code="customUri.required.content.p2" bundle="login"}}
```